server/src/internalClusterTest/java/org/opensearch/search/functionscore/FunctionScoreFieldValueIT.java

```diff
@@ -99,7 +99,7 @@ public void testFieldValueFactor() throws IOException {
         client().prepareIndex("test").setId("1").setSource("test", 5, "body", "foo").get();
         client().prepareIndex("test").setId("2").setSource("test", 17, "body", "foo").get();
         client().prepareIndex("test").setId("3").setSource("body", "bar").get();
-
+        indexRandomForConcurrentSearch("test");
```

Comment:
should this be after `refresh()` ?  Such that actual documents are inserted in a segment and then these random docs get added to a new segment

Reply:
Yes they should, good catch. Let me fix all instances of this.